src/main/java/org/humancellatlas/ingest/model/SubmissionEnvelopeReference.java

```diff
@@ -17,5 +19,6 @@
 public class SubmissionEnvelopeReference {
     private final String id;
     private final String uuid;
+    private final String state;
```

Comment:
Why String and not the Enum?

Reply:
Yep, i think I could convert right away and use the enum here.

Reply:
done!